admin-frontend/open_admin_app/lib/widgets/user/list/list_users_bloc.dart

```diff
@@ -37,7 +43,13 @@ class ListUsersBloc implements Bloc {
     Timer(const Duration(milliseconds: 300), () {
       if (newSearch == search) {
         // hasn't changed
-        _requestSearch(); // don't need to await it, async is fine
+        if(isPerson) {
+          _requestSearch();
+        }
```

Comment:
the only difference should be the personType, are you sure you don't want to parameterise requestSearch?